html-templates/verified_credentials.html

```diff
@@ -289,9 +290,19 @@ <h5 v-if="state.showScanned" class="fw-bolder mb-3">
 
   <script type="text/javascript">
     const { createApp, ref, reactive, computed, watch } = Vue;
-    /**
+   /**
      * @typedef  {"intro" |"verified" | "failed" | "pending"| "expired" |"abandoned"} AppStates
      */
+    // user agent detection
+   const uap = new UAParser();
+   const getBrowserUA = () => {
+     PREFIX_MOBILE  = 'Mobile ';
+     // @type string
+     const b = uap.getResult().browser.name
+     const browserName = b.startsWith(PREFIX_MOBILE) ? b.split(" ")[1] : b.split(" ")[0]
+     return `{{wallet_deep_link}}?browser=${browserName.toUpperCase()}`;
```

Comment:
Possible cause of some of the issues: we should be appending `&browser=...` since the deep link payload is already a query parameter and the `?`defining the beginning of the query params namespace  is already present in the url

Reply:
I'll give that a try. Good idea